pkg/standalone/run.go

```diff
@@ -238,10 +243,32 @@ func (config *RunConfig) getArgs() []string {
 	args := []string{}
 
 	schema := reflect.ValueOf(*config)
+	args = getArgsFromSchema(schema, args)
+
+	if config.ConfigFile != "" {
+		sentryAddress := mtlsEndpoint(config.ConfigFile)
+		if sentryAddress != "" {
+			// mTLS is enabled locally, set it up.
+			args = append(args, "--enable-mtls", "--sentry-address", sentryAddress)
+		}
+	}
+
+	if print.IsJSONLogEnabled() {
+		args = append(args, "--log-as-json")
+	}
+	return args
+}
+
+// Recursive function to get all the args from the config struct.
+// this is needed because the config struct has embedded struct.
+func getArgsFromSchema(schema reflect.Value, args []string) []string {
 	for i := 0; i < schema.NumField(); i++ {
 		valueField := schema.Field(i).Interface()
 		typeField := schema.Type().Field(i)
 		key := typeField.Tag.Get("arg")
+		if typeField.Type.Kind() == reflect.Struct {
+			args = getArgsFromSchema(schema.Field(i), args)
```

Comment:
should this be 
```suggestion
			args = append(args, getArgsFromSchema(schema.Field(i), args)) 
```
? similar to lines 282 and 287?

We are losing the args that was constructed before if it sees a struct type field here?

Reply:
no, this is not needed as we are passing back the accumulated args in recursive function. In fact we need a `continue` after this `args = getArgsFromSchema(schema.Field(i), args)`. Currently `continue` is there in the next `if block` but we should continue from here only.